spec/services/collators/regular_outgoings_collator_spec.rb

```diff
@@ -26,7 +26,6 @@
         collator
         disposable_income_summary.reload
         expect(disposable_income_summary).to have_attributes(
```

Comment:
@starswan Here we are removing "child_care_all_sources" from response object which can cause issue at the client application(s). Or client is aware of this change?